src/js/controller/LayersListController.js

```diff
@@ -152,8 +189,10 @@
     } else if (action == 'merge') {
       this.mergeDownCurrentLayer_();
     } else if (action == 'edit') {
-      this.renameCurrentLayer_();
+      this.openCurrentLayerRenameForm_();
+      return;
     }
+    this.renderLayerList_();
```

Comment:
Why is this needed? 
- moveLayerUp triggers a reset
- moveLayerDown triggers a reset
- createKayer triggers a reset
- removeCurrentLayer triggers a reset
- mergeDown... triggers a reset (and also calls renderLayersList manually which is not needed)